src/cli/commands/bin.js

```diff
@@ -19,6 +19,6 @@ export function run(
     config.registries[RegistryYarn.registry].folder,
     '.bin',
   );
-  console.log(binFolder);
+  reporter.log(binFolder);
```

Comment:
Oh ok, interesting. Is there a good way to test the console output of this command without using the `reporter`?